src/main/java/com/fundation/webservice/model/CriteriaConvert.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+/**
+ * Implements the model class File and the getter and setter´s methods
```

Comment:
It would be better if we provide a customized description rather than the generic one from this example.